lib/management.js

```diff
@@ -1,3 +1,4 @@
+const { URL } = require('url');
```

Comment:
I don't think `url.URL` is supported in Node v6. Instead there is a property: `url.Url` but it doesn't seem to work as intended.

Based on this blog: https://www.contentful.com/blog/2016/11/02/node-v7-urls-deprecations-warnings-and-developer-experience/
Looks like support was added in v7?

Reply:
https://nodejs.org/docs/latest-v6.x/api/url.html#url_the_whatwg_url_api
it is also added in 6.13